ArdusatSDK.h

```diff
@@ -436,6 +439,10 @@ class RGBLightTCS: public RGBLight {
  * @brief Encapsulates all functionality related to the ISL29125 RGBLight Sensor
  */
 class RGBLightISL: public RGBLight {
```

Comment:
I think the `configure` methods belong on these sensor-specific classes, not on the generic `RGBLight` class.